style.css

```diff
@@ -0,0 +1,167 @@
+/* removes text under-line */
+A {
+  text-decoration: none;
+}
+
+
+.nav-bar {
+    display: flex;
```

Comment:
You are repeating a lot of `display: flex` properties throughout your CSS. Please, make a `flex` class with this property and apply it in the 'flexed' elements you want, this will make your CSS a lot more readable. Keep this in mind for future projects :wink: